shared/features/navigation/primary-navigation/_translations/primary-navigation.en.json

```
@@ -4,7 +4,7 @@
     "financial": "Financial",
     "program": "Program",
     "projects": "Explore Project",
-    "maintainer": "Maintainer"
+    "manageProject": "Manage Project"
```

```suggestion
    "manageProject": "Manage Projects"
```

No ? Same for the line above.